api/data_explorer/controllers/facets_controller.py

```diff
@@ -1,37 +1,54 @@
-from elasticsearch import Elasticsearch
-from elasticsearch_dsl import HistogramFacet
-from elasticsearch_dsl import TermsFacet
-from flask import current_app
-
 from data_explorer.models.facet import Facet
 from data_explorer.models.facet_value import FacetValue
 from data_explorer.models.facets_response import FacetsResponse
+from elasticsearch_dsl import HistogramFacet
+from flask import current_app
+
 from ..dataset_faceted_search import DatasetFacetedSearch
+import urllib
 
 
-def facets_get():
-    """
-    Returns facets.
+def facets_get(filter=None):  # noqa: E501
```

Comment:
filterArr to match pydoc

Reply:
This parameter name actually comes from the API spec. I want to change this anyway, since `filter` shadows a built-in python name, but I'd like to reserve this for a separate PR since it will involve renaming in the api.yaml and regenerating all of the specs, which would bury the other code in this PR.

In the meantime, changed the pydoc to reference `filter`.